src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1061,6 +1061,11 @@ public void SarifTransformerTests_ToCurrent_BasicResult()
         {
           ""ruleId"": ""C2001"",
           ""level"": ""error"",
+          ""message"": {
+            ""arguments"": [
+              ""ptr""
+            ]
+          },
```

Comment:
This is correct. The spec now explicitly addresses this case in : §3.19.7, "ruleMessageId property":

>If the message string identified by `ruleId` and `ruleMessageId` includes placeholders (§3.9.4), then `result.message.arguments` (§3.19.6, §3.9.11) **SHALL** contain the replacement values for the placeholders. In this situation, `result.message` will contain _only_ the `arguments` property.
>
> EXAMPLE 2: In this example, the message string identified by `ruleId` and `ruleMessageId` has a single placeholder `"{0}"`. `message.arguments holds` the replacement value `"counter"`.
```
{                                # A run object (§3.11).
  "results": [
    {                            # A result object (§3.19).
      "ruleId": "CA2101",
      "ruleMessageId": "default",
      "message": {
        "arguments": [
          "counter"
        ]
      }
    }
  ],

  "resources": {
    "rules": {
      "CA2101": {                # A rule object
        "messageStrings": {
          "default": "Variable \"{0}\" is uninitialized."
        }
      }
    }
  }
}
```

Reply:
Great thank you.